etc/profile-a-l/google-chrome-beta.profile

```diff
@@ -8,10 +8,16 @@ include globals.local
 noblacklist ${HOME}/.cache/google-chrome-beta
 noblacklist ${HOME}/.config/google-chrome-beta
 
+noblacklist ${HOME}/.config/chrome-beta-flags.conf
```

Comment:
We should keep the file names of the flags configuration cfr. what you used in google-chrome.profile. I'm pretty sure the beta version will otherwise not be able to find those (even when they exist). So just drop the '-beta' part. Same goes for the whitelists and the unstable profile. I'll mark those as well, just so you know where to edit.

Reply:
I've tested the profiles as they are now and google-chrome-beta does seem to look for `chrome-beta-flags.conf / chrome-beta-flags.config`.